doc/intro.md

```diff
@@ -75,10 +72,44 @@ where:
 
 This method involves projecting births, deaths and migrations by age and sex.
 
-See below the minimal cohort component method we aim to provide:
+See below the cohort component method we aim to provide:
 
 ![minimal CCM](images/MVP_generic_CCM_flowchart.png)
 
+## User-defined parameters for the model
+
+Some parameters in the model can be specified by the user. The parameters, with defaults in parentheses, are:
+
+* First year of projection (2015)
+* Last year of projection (2018)
+* Start & end years of year range to average over when projecting deaths (2010-2014)
+* Start & end years of year range to average over when projecting domestic migration (2003-2014)
```

Comment:
needs updating to start and end years for each in and out types of domestic migration